### PR TITLE
[zh] Resync VAP reference page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 1]
```diff
@@ -274,6 +275,7 @@ The parameter resource could be as following:
 
 <!--
 This policy parameter resource limits deployments to a max of 3 replicas.
+
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:278]: Do we also need to add a blank line to the zh para?

**Review by windsonsea — COMMENTED** [review]
See [preview](https://deploy-preview-47925--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/access-authn-authz/validating-admission-policy/).

/approve

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:278]: It should be fine.

**Pull request merged**